esp-wifi/src/tasks.rs

```diff
@@ -33,14 +33,14 @@ pub extern "C" fn worker_task3() {
 
 pub extern "C" fn worker_task2() {
     loop {
-        let mut to_run = SimpleQueue::<_, 20>::new();
+        let mut to_run = SimpleQueue::<_, 40>::new();
```

Comment:
This should be unnecessary. We should just stop checking timers if this queue fills up, and we should loop back.